### PR TITLE
Feature/1731 case flag coverage

### PR DESCRIPTION
**Pull request by velmurugan-mani**
### JIRA link (if applicable) ###
[1731 – feature/case-flags-2-1-cui-ra](https://sonarcloud.io/summary/new_code?id=prl-cos-api&pullRequest=1731)

### Change description ###
Sonar Test Coverage

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/mapper/citizen/CaseDataApplicantElementsMapper.java [hunk 1]
```diff
@@ -200,12 +199,6 @@ private static List<Element<ChildrenAndApplicantRelation>> buildChildAndApplican
...
 
     private static void checkAndupdatePcqIdForParty(PartyDetails party, String applicantPcqId) {
-        if (ObjectUtils.isNotEmpty(party.getUser())) {
-            if (StringUtils.isEmpty(party.getUser().getPcqId())) {
-                party.getUser().setPcqId(applicantPcqId);
-            }
-        } else {
-            party.setUser(User.builder().pcqId(applicantPcqId).build());
-        }
+        party.setUser(User.builder().pcqId(applicantPcqId).build());
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/mapper/citizen/CaseDataApplicantElementsMapper.java:202]: Explain this code change please?

**Comment by AlokDatta** [comment]
There are code conflicts and 15 new code smells in this PR itself. fixing or breaking?

**Comment by velmurugan-mani** [comment]
close for sonar test

**Comment by velmurugan-mani** [comment]
for sonar test build update

File: src/main/java/uk/gov/hmcts/reform/prl/mapper/citizen/CaseDataApplicantElementsMapper.java (hunk 1 above)
Comment (velmurugan-mani) [reply src/main/java/uk/gov/hmcts/reform/prl/mapper/citizen/CaseDataApplicantElementsMapper.java:202]: This code is required for sonar code coverage and has already been discussed with Sairam.

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**